sdks/python/apache_beam/coders/coders_test_common.py

```diff
@@ -80,8 +81,9 @@ def tearDownClass(cls):
         coders.RunnerAPICoderHolder,
         coders.ToBytesCoder
     ])
-    assert not standard - cls.seen, standard - cls.seen
-    assert not standard - cls.seen_nested, standard - cls.seen_nested
+    cls.seen_nested -= set([coders.ProtoCoder, CustomCoder])
+    assert not standard - cls.seen
+    assert not cls.seen_nested - standard
```

Comment:
Why is this change necessary?

Reply:
The major change here is `assert not cls.seen_nested - standard`.  In my case I found that there is a `TimerCoder` in `standard` set, which is expected, but not in `cls.seen_nested`. I checked the code  https://github.com/apache/beam/blob/b3e06126405105836f9ae15a15d5e1c7189b7f6e/sdks/python/apache_beam/coders/coders_test_common.py#L93, only when the input coder is a `TupleCoder`, there are new coders added into `cls.seen_nested`, which means, `cls.seen_nested` should be a subset of `standard`, correct? @robertwb What do you think?